geode/dispatcher.py

```diff
@@ -295,9 +310,9 @@ def distance_matrix(self, origins, destinations, max_meters=MAX_METERS, provider
             self.distance_matrix_with_session(origins, destinations, max_meters, provider=provider, return_inverse=return_inverse)
         )
 
-    def distance_pairs(self, origins, destinations, max_meters=MAX_METERS, provider=None):
+    def distance_pairs(self, origins, destinations, max_meters=MAX_METERS, provider=None, return_inverse=False):
```

Comment:
Why are we passing hard-coded values here?

Reply:
which values? False is just optional default